### PR TITLE
Add support for Account Public Key data type

### PR DESCRIPTION
**Pull request by intelliot**
Per https://xrpl.org/base58-encodings.html

### CODE REVIEW

File: src/xrp-codec.test.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ function toHex(bytes: Buffer) {
 }
 
 function toBytes(hex: string) {
-  return Buffer.from(hex, 'hex').toJSON().data
```
Comment (intelliot) [inline src/xrp-codec.test.ts]: I don't know why `.toJSON().data` was here. It does not seem to make any difference.

**Comment by hbergren** [comment]
Implementation seems reasonable.

Is it worth adding a test for `encodeAccountPublic` or `decodeAccountPublic`?

**Comment by intelliot** [comment]
Yes, test is here :) https://github.com/ripple/ripple-address-codec/pull/35/files#diff-56fdceac77c16bd6de60b8811ab96d68R37

**Comment by intelliot** [comment]
The test vector is from https://xrpl.org/use-payment-channels.html

**Pull request merged**